### PR TITLE
Add russian i18n

### PR DESCRIPTION
**Pull request by isRuslan**
Add files for `{appname} -l ru` workshopper language :ru:

### CODE REVIEW

**Comment by martinheidegger** [comment]
@isRuslan Do you have another person that can proofread that?

**Comment by isRuslan** [comment]
Russian-speaking people on nodeschool?
Maybe @sudodoki will check that.

**Comment by rvagg** [comment]
@sidorares would probably be helpful I'm guessing

**Comment by sidorares** [comment]
yes, I was actually going to start translating

**Comment by rvagg** [comment]
Well, now there are two of you to check each others work!

File: i18n/footer/ru.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ __»__ Для вывода описнаия задачи, введите: `{appname} print`
```
Comment (sidorares) [inline i18n/footer/ru.md]: описнаия -> описания

**Comment by sidorares** [comment]
:+1:  (pending one typo)

**Comment by rvagg** [comment]
added you both as collaborators so you can keep this updated, @isRuslan you should feel free to merge this, then ping me when you're ready for a release and I'll push one out.

**Comment by isRuslan** [comment]
@rvagg thanks, typo fixed.

I can also help to translate https://github.com/rvagg/learnyounode @sidorares

**Comment by rvagg** [comment]
so maybe I'll hold off on publishing this until we have something to start with in learnyounode?

**Comment by isRuslan** [comment]
Sure, I can send "work in progress: russian translation" pull request to [learyounode/pulls](https://github.com/rvagg/learnyounode/pulls) where all work about ru-i18n will be.
Then I can add @sidorares to my fork as collaborator - we both could work in the same PR.

**Pull request merged**